app/assets/v2/js/pages/dashboard.js

```diff
@@ -108,7 +108,7 @@ function debounce(func, wait, immediate) {
   };
 }
 
-const scrub = value => value.replace(/[\W]+/g, '');
+const scrub = value => value.replace(/[!@#$%^&*(),.?":{}|<>]+/g, '');
```

Comment:
I'm wondering if this opens us up at all to security risks and what the minimum characters we should allow is - I'm thinking `+` and `#` - any other weirdness out there y'all can think of?

Reply:
nope ! just those hence added those in
@octavioamu  ?

Reply:
lgtm!